example/examples/NativeWebpage.tsx

```diff
@@ -11,10 +11,13 @@ export default class NativeWebpage extends Component<Props, State> {
 
   render() {
     return (
-      <View style={{height: 400}}>
+      <View style={{ height: 400 }}>
         <WebView
-          source={{uri: 'https://infinite.red'}}
-          style={{width: '100%', height: '100%'}}
+          source={{
```

Comment:
Are these changes to the example intentional? I think we'll want to keep the sample as it was.

Reply:
It was due to my prettier. Maybe the author haven’t run prettier before on this file.

Reply:
It looks like the url has been changed and your prop additional is still there. We might want to revert those!

Reply:
Oops, I forgot to remove that one. Thanks for your note.

Reply:
Reverted.